### DISCUSSION TITLE
Unicode rendering missing character in Linux simulator

### DISCUSSION THREAD

**Discussion by paulmelis** [StereoKit/StereoKit#905] [General]
## Description

I'm using a `Text` that contains a `▲` character. This shows up correctly when deployed to an ML2, on PC VR with a Varjo XR-4 and in the simulator on Windows. But it shows up as `?` in the simulator on Linux, while I seem to remember it displaying correctly a long time ago.

## Platform / Environment

0.3.9, with the sk-multi template

**Comment by maluoi** [comment]
This happens when the current font and font fallbacks don't include a glyph for the unicode symbol! On Linux, StereoKit pulls the system's default sans-serif font, which can vary a fair bit from distro to distro? The solution for now would be to pick and assign a font that does include the glyph.

I'd love to explore a better font/glyph solution that will be more font family name oriented, like how CSS does it. That would allow for more robust glyph fallback. Unfortunately it needs to be done separately per-OS, since each has different font APIs, some of which are pretty arcane.

**Comment by paulmelis** [comment]
Okay, this makes sense. Perhaps change this into a discussion item, as it isn't a bug then?
